### ISSUE TITLE
Evasion tactics [WARNING]

### ISSUE THREAD

**Issue by JPShag** [juzeon/SydneyQt#257]
The compiled release will not run in a sandbox environment, careful with this one.

**Comment by juzeon** [comment]
The releases are all compiled by [GitHub Actions](https://github.com/dAppServer/wails-build-action) from the source code. If you don't trust the releases, you can compile it from the source.

We do not intentionally test compatibility for the sandbox environment. This may be more of a [Wails](https://github.com/wailsapp/wails) issue.

**Comment by JPShag** [comment]
Yes, thank you for your response. I don't mean to accuse. Just commonplace in anti-reversing. I will see if they can shed light.

**Comment by JPShag** [comment]
Waiting for [leeanthony](https://github.com/wailsapp/wails/issues/3689)
